APTITUDE/APTITUDETEST.Core/ViewModels/UserImportVM.cs

```diff
@@ -5,5 +5,7 @@ public class UserImportVM
         public string firstname { get; set; }
         public string email { get; set; }
         public long contactnumber { get; set; }
+        public int? groupid { get; set; } = 0;
+        public int? collegeid { get; set; } = 0;
     }
```

Comment:
Remove this two fields as we are using same groupid and collageid for each user records.